examples/todo-app/src/todo-app.ts

```diff
@@ -0,0 +1,14 @@
+import { FASTElement } from "@microsoft/fast-element";
+import { styles } from "./todo-app.styles.js";
+import { template } from "./todo-app.template.js";
+import { TodoList } from "./todo-list.js";
+
+export class TodoApp extends FASTElement {
+    @TodoList todos!: TodoList;
+}
+
+export const app = FASTElement.metadata(TodoApp, {
```

Comment:
Not related to this PR per-se, but the `metadata` method read a bit oddly to me here. I think maybe because I generally think of metadata as properties of a thing, and not an action to take on a thing (a method). I wonder if this method would communicate the constructive nature of the behavior better if it were named `compose`, `create`, `associate`, or similar?

Reply:
I wasn't really sure what to call this TBH.

Reply:
Yep, just a thought - feel free to take it, leave it, or change it.

Reply:
The more I think about it, I think `compose` the what we want here. I may see if I can make that update as part of this real quick.